ingest/models.py

```diff
@@ -259,4 +263,55 @@ def __str__(self):
     r24_name = models.CharField(max_length=256)
     r24_directory = models.CharField(max_length=256)
     
+class BilUser(models.Model):
+
+    name = models.CharField(max_length=256)
+    orcid = models.CharField(max_length=256)
+    affiliation = models.CharField(max_length=256)
+    affiliation_identifier = models.CharField(max_length=256)
+    auth_user_id = models.ForeignKey(auth_user, on_delete=models.SET_NULL, null = True, blank=True)
+
+class Project(models.Model):
+    
+    project_pi = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, blank=True)
+    funded_by = models.CharField(max_length=256)
+    is_biccn = models.BooleanField(default=False)
+    
+class ProjectCollections(models.Model):
+    def __str__(self):
+        return self.name
+    project_id = models.ForeignKey(Project, on_delete=models.SET_NULL, null=True, blank=True)
+    collection_id = models.ForeignKey(Collection, on_delete=models.SET_NULL, null=True, blank=True)
+
+class ProjectUsers(models.Model):
+ 
+    project_id = models.ForeignKey(Project, on_delete=models.SET_NULL, null=True, blank=True)
+    bil_user_id = models.ForeignKey(BilUser, on_delete=models.SET_NULL, null = True, blank=True)
```

Comment:
i think leaving this as on_delete=models.SET_NULL makes sense, because if we remove a bil_user, we don't want to lose their data